js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js

```diff
@@ -340,13 +313,14 @@
         });
 
         arangoHelper.initSigma();
-      }.bind(this);
+      }).bind(this);
 
-      $(window).resize(function () {
+      // TODO: Should a UI function be part of the router?
+      $(window).on('resize', function () {
         self.handleResize();
       });
 
-      $(window).scroll(function () {
+      $(window).on('scroll', function () {
         // self.handleScroll()
       });
```

Comment:
```suggestion
```

Reply:
Can be removed completely, as it does not execute any code.

Reply:
Can also remove the handler? I don't think it is used for anything else.